.changeset/tender-dodos-trade.md

```diff
@@ -1,7 +0,0 @@
----
```

Comment:
This changeset should have been deleted in the last merge of "Version Packages", but I think the github action failed to run for this PR.